feature_extraction/compare_features.ipynb

```diff
@@ -0,0 +1,278 @@
+{
```

Comment:
<p>I really like the idea of investigating the overlapping features. However I am a bit concerned on the validity of this chi sqaured test. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/treyduffus/multiclass_lungcancer_classification_models/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>